scripts/screen_record.sh

```diff
@@ -54,15 +49,17 @@ stopRecording() {
 
     pkill -SIGINT -f wf-recorder
 
-    recentFile=$(ls -t "$outputDir"/recording_*.mp4 | head -n 1)
+    recentFile=$(ls -t "$tempDir"/*.mp4 | head -n 1)
```

Comment:
Let's make the search a bit unique, as we don't want to just pickup any .mp4 file there is.

We could just always use the same filename and prevent two recordings from happening at the same time